### DISCUSSION TITLE
packaged python app not working well

### DISCUSSION THREAD

**Discussion by tomererez1998** [pyinstaller/pyinstaller#8287] [Help]
Hey, I have a bug when building a python 3.11.6 app into an exe.
before building it works perfectly as a .py file.
after building, it gets to the point where the neural network(yolo) needs to predict and restarts the instance and I see the prints in the cmd again and again:
(app started, about to predict…):

App started
before opening stream
before prediction with nnet

App started
before opening stream
before prediction with nnet

App started
before opening stream
before prediction with nnet

the app does not pass this point and keep on being stuck in this loop
i tried using try except and printing the exception but it does not catch the exception and it is stuck.
Who can I contact about that
Best regards/Mit freundlichen Grüßen,

Tomer Erez

**Comment by bwoodsend** [comment]
https://pyinstaller.org/en/latest/common-issues-and-pitfalls.html#multi-processing
